rasa/cli/evaluate.py

```diff
@@ -83,7 +87,7 @@ def _run_markers_cli(args: argparse.Namespace) -> None:
         args: The arguments passed in from the CLI.
     """
     seed = args.seed if "seed" in args else None
-    count = args.count if "count" in args else None
+    count = args.count if "N" in args else None
```

Comment:
This won't work because `count` doesn't exist in `args` any more. I would add `dest='count'` to the argument definition and then leave it as it was before - since I think having `N` as an entry in the argument namespace will be a lot less clear

Reply:
Ok will fix it.

Reply:
What I meant by this was to add the `dest` to the `add_argument` calls for `N` since then you can redirect where the value is stored (since in the code imo `count` is more understandable) - this also means that none of the rest of the code needs changed because the value the user gives for `N` would be saved to `args.count`

Reply:
I have reverted back from `N` to `count` since it's clearer what it means. 

>I would add dest='count' to the argument definition

Where did you want to add that? `count` is already the dest [here](https://github.com/RasaHQ/rasa/blob/30026d2658f2dd079c3d76e93eeee1cc24e8f633/rasa/cli/arguments/evaluate.py#L54) and [here](https://github.com/RasaHQ/rasa/blob/30026d2658f2dd079c3d76e93eeee1cc24e8f633/rasa/cli/arguments/evaluate.py#L64) (first positional argument is [dest](https://docs.python.org/3/library/argparse.html#dest)).

Reply:
First postional argument is `name` (see [here](https://docs.python.org/3/library/argparse.html#the-add-argument-method) - `dest` is set as `name` by default unless `dest` is overwritten as it says in the docs you've linked. So what I was suggesting was change `name` to `N`, but then set `dest` as `count` (as an additional named argument) to avoid breaking the usage of `args.count` (since `count` will be where the value the user gives for `N` is saved)

Reply:
Ok thanks for clarifying. Are you happy to keep it as `count` everywhere? Or do you think it's better to implement the change you suggested? What I mean is: what do you think will be clearer for users, `count` or `N`?

Reply:
I'm happy to keep it as `count`, I thought you were keen to change it to `n` for clarity (I personally am not sure there's a difference between the two in terms of clarity) - if you want to change it to `n` I'm happy with that too

Reply:
Thanks Matt. I'm happy to keep it as `count`. 
Let me know if you're happy with the rest of the PR.